File: app/src/main/java/com/coderstory/flyme/patchModule/IsEnable.java
package com.coderstory.flyme.patchModule;


import com.coderstory.flyme.tools.XposedHelper;
import com.coderstory.flyme.xposed.IModule;

import de.robv.android.xposed.IXposedHookZygoteInit;
import de.robv.android.xposed.XC_MethodReplacement;
import de.robv.android.xposed.callbacks.XC_InitPackageResources;
import de.robv.android.xposed.callbacks.XC_LoadPackage;

public class IsEnable extends XposedHelper implements IModule {


    @Override
    public void handleInitPackageResources(XC_InitPackageResources.InitPackageResourcesParam resparam) {
    }

    @Override
    public void handleLoadPackage(XC_LoadPackage.LoadPackageParam lpparam) {
        if (lpparam.packageName.equals("com.coderstory.flyme")) {
            findAndHookMethod("com.coderstory.flyme.activity.MainActivity", lpparam.classLoader, "isEnable", XC_MethodReplacement.returnConstant(true));
        }
    }

    @Override
    public void initZygote(IXposedHookZygoteInit.StartupParam paramStartupParam) {
    }

}

File: app/src/main/java/com/coderstory/flyme/patchModule/IsEnable.kt
package com.coderstory.flyme.patchModule


import com.coderstory.flyme.tools.XposedHelper
import com.coderstory.flyme.xposed.IModule
import de.robv.android.xposed.IXposedHookZygoteInit.StartupParam
import de.robv.android.xposed.XC_MethodReplacement
import de.robv.android.xposed.callbacks.XC_InitPackageResources.InitPackageResourcesParam
import de.robv.android.xposed.callbacks.XC_LoadPackage.LoadPackageParam

class IsEnable : XposedHelper(), IModule {
    override fun handleInitPackageResources(respray: InitPackageResourcesParam) {}
    override fun handleLoadPackage(param: LoadPackageParam) {
        if (param.packageName == "com.coderstory.flyme") {
            XposedHelper.Companion.findAndHookMethod("com.coderstory.flyme.activity.MainActivity", param.classLoader, "isEnable", XC_MethodReplacement.returnConstant(true))
        }
    }

    override fun initZygote(paramStartupParam: StartupParam?) {}
}